### DISCUSSION TITLE
How do you use elastix?

### DISCUSSION THREAD

**Discussion by ntatsisk** [SuperElastix/elastix#893] [Ideas]
It would be helpful if people share info related to their application of elastix. For example, such knowledge can be directly incorporated in [the  collection of jupyter notebooks](https://github.com/InsightSoftwareConsortium/ITKElastix/tree/main/examples) and the [model zoo of parameter files](https://elastix.lumc.nl/modelzoo/) benefiting the rest of the community!

So, **how do you use elastix?**

Additionally, you can comment on:
* elastix flavor: executable, C++ API, [ITK-Elastix](https://github.com/InsightSoftwareConsortium/ITKElastix)/other wrapping,[ elastix-napari](https://github.com/SuperElastix/elastix-napari) napari plugin, [SlicerElastix](https://github.com/lassoan/SlicerElastix) 3D Slicer plugin, etc
* dataset: private or public
* result visualization
* availability of shareable code/notebook/parameter file/tips & tricks
* anything else really

Feel free to reply below. Thanks! 🙏

**Comment by cnmy-ro** [comment]
I'm using elastix as a tool in my MRI reconstruction project mainly to perform cross-contrast registration on clinical MRI scans of the brain. My research focuses on multi-contrast reconstruction which requires the contrasts to be perfectly aligned. The aforementioned registration step is used to correct for any inter-scan motion.

- Elastix flavor: ITK-Elastix, because my workflow is entirely in Python.
- Dataset: Private
- Results, notebooks, tips & tricks: I'm still in the beginner stage in exploring elastix's capabilities in my project. I hope to share some of these contents soon!
